ablateCore/particles/particles.c

```diff
@@ -163,17 +167,18 @@ PetscErrorCode ParticleView(ParticleData particleData, PetscViewer viewer) {
     PetscFunctionReturn(0);
 }
 
-PetscErrorCode ParticleViewFromOptions(ParticleData particleData, char *optionname) {
+PetscErrorCode ParticleViewFromOptions(ParticleData particleData,PetscObject obj, char *title) {
     PetscFunctionBegin;
+    Vec            particleVector;
     PetscErrorCode ierr;
-    PetscViewer viewer;
```

Comment:
I think that you are introducing old code here.  I think that putting back this old code may result in some bugs.